ros/src/computing/perception/detection/packages/lidar_tracker/CMakeLists.txt

```diff
@@ -167,35 +172,6 @@ if(EXISTS "/usr/local/cuda")
   endif()
 
   #QT Stuff
-  # set(CMAKE_AUTOMOC ON)
-  set(CMAKE_INCLUDE_CURRENT_DIR ON)
-
-  add_custom_command(
-    OUTPUT ui_mainwindow.h
-    COMMAND
-    ${Qt5BIN}/uic
-    "-o" "${CMAKE_CURRENT_SOURCE_DIR}/nodes/vscan_lidar_track/ui_mainwindow.h"
-    "${CMAKE_CURRENT_SOURCE_DIR}/nodes/vscan_lidar_track/mainwindow.ui"
-    )
-  add_custom_target(vehicle_tracker_ui_mainwindow DEPENDS ui_mainwindow.h)
-
-  add_custom_command(
-    OUTPUT moc_mainwindow.cpp
-    COMMAND
-    ${Qt5BIN}/moc
-    "-o" "${CMAKE_CURRENT_SOURCE_DIR}/nodes/vscan_lidar_track/moc_mainwindow.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/nodes/vscan_lidar_track/mainwindow.h"
-    )
-  add_custom_target(vehicle_tracker_moc_mainwindow DEPENDS moc_mainwindow.cpp)
-
-  add_custom_command(
-    OUTPUT moc_rbsspfvehicletracker.cpp
-    COMMAND
-    ${Qt5BIN}/moc
-    "-o" "${CMAKE_CURRENT_SOURCE_DIR}/nodes/vscan_lidar_track/moc_rbsspfvehicletracker.cpp"
-    "${CMAKE_CURRENT_SOURCE_DIR}/nodes/vscan_lidar_track/rbsspfvehicletracker.h"
```

Comment:
I wonder why you removed this part? This file for instance is still in the repo: https://github.com/CPFL/Autoware/blob/develop/ros/src/computing/perception/detection/packages/lidar_tracker/nodes/vscan_lidar_track/rbsspfvehicletracker.h.

Reply:
Never mind, I just saw that you actually included them below.